bower.json.erb

```diff
@@ -2,7 +2,7 @@
   "name": "rad-consumer",
   "dependencies": {
     "dough": "<%= gem_path('dough-ruby') %>",
-    "yeast": "git://github.com/moneyadviceservice/yeast.git#master",
+    "yeast": "git://github.com/moneyadviceservice/yeast.git#a1dad29959e057aae926f92d61b55dd519bed812",
```

Comment:
Fair enough

> On 15 Mar 2016, at 09:24, Peter Aitken <notifications@github.com> wrote:
> 
> In bower.json.erb:
> 
> > @@ -2,7 +2,7 @@
> >    "name": "rad-consumer",
> >    "dependencies": {
> >      "dough": "<%= gem_path('dough-ruby') %>",
> > -    "yeast": "git://github.com/moneyadviceservice/yeast.git#master",
> > +    "yeast": "git://github.com/moneyadviceservice/yeast.git#a1dad29959e057aae926f92d61b55dd519bed812",
> Yeast master has had render_content move out to Dough.
> Our version of Dough is over a year old and didn't want to open a can of worms there so pinned the version of Yeast.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/moneyadviceservice/rad_consumer/pull/281/files#r56134310
>